.gitignore

```diff
@@ -1,83 +1,86 @@
-# suppress compiler/linker output

```

Comment:
diff looks bad due to line ending changes. But you can select "hide whitespace changes" in the gh diff settings.

Reply:
ugh they were `\r\n`!?

btw. I added a separate etc/.gitignore
now it's mixed up where the stuff is ignored that is created in etc/ :-)

Reply:
I don't like multiple .gitignores tbh

Reply:
fine by me